pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -155,6 +155,13 @@ func runVersionUpgrade(ctx context.Context, t test.Test, c cluster.Cluster) {
 	mvt.InMixedVersion(
 		"test schema change step",
 		func(ctx context.Context, l *logger.Logger, rng *rand.Rand, h *mixedversion.Helper) error {
+			// The schema change workload is only compatible with the branch it was built
+			// from and the major version before that.
+			if h.IsSkipVersionUpgrade() {
+				l.Printf("skipping: upgrade is a version skip")
```

Comment:
Nit: "version skip upgrades are unsupported"

Reply:
Switched to "schema change workload is unsupported with skip version upgrades"